tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/TemplatesTest.cs

```diff
@@ -1481,12 +1481,21 @@ public void TestCustomFunction2()
         public void TestInjectLG()
         {
             var templates = Templates.ParseFile(GetExampleFilePath("./InjectionTest/inject.lg"));
-            
-            var (evaled, error) = Expression.Parse("foo.bar()").TryEvaluate(null);
-            
+
```

Comment:
Maybe add more tests about the "Expression.TryEvaluate(IMemory state)" is better. 
for example:
```
var memory = StackedMemory.Wrap(...);
```
or 
```
var memory = new StackMemoory();
memory.push(new SimpleMemory(xxx))
memory.push(new CustomizedMeoory(xxx))
```

Reply:
tests added